### ISSUE TITLE
When I have nested function with return expr in a fixture, PT005 false positive happens

### ISSUE THREAD

**Issue by afonasev** [m-burst/flake8-pytest-style#52]
# Bug report

## What's wrong

```python
@pytest.fixture()
def _any_fixture(mocker):  # PT005 fixture '_any_fixture' returns a value, remove leading underscore
    def nested_function():
        return 1

    mocker.patch('...', nested_function)
```

## How it should work

PT005 error should not happen.

## System information

* Operating system: macOS
* Python version: 3.8
* flake8 version: 3.8.3
* flake8-pytest-style version: 0.1.3

**Comment by m-burst** [comment]
Hi @afonasev,
Thanks for your report!
I have just released v1.2.2 which fixes the problem.
